src/test/java/com/kainos/ea/integration/CapabilityIntegrationTest.java

```diff
@@ -20,13 +28,31 @@ public class CapabilityIntegrationTest {
             new ResourceConfigurationSourceProvider()
     );
 
+    String validToken;
+
+    @BeforeEach
+    public void init() {
+        String fakeSecret = System.getenv("JWT_SECRET");
```

Comment:
Is this really fake if it's using the `JWT_SECRET` environment variable 🤔

Reply:
All fixed :)